### ISSUE TITLE
macos: Show scroll bars: always, automatic might be worth picking up from the OS

### ISSUE THREAD

**Issue by vlsi** [weisJ/darklaf#145]
https://twitter.com/JamesIvings/status/1252841057160577027

I've not investigated, but it might be worth synchronizing this setting from the OS as well.

**Comment by weisJ** [comment]
What does this setting effectively do? If there is no scrollbar necessary will it show one nonetheless?

**Comment by hannesbraun** [comment]
No. If there is no scrollbar necessary,  no scrollbar will be shown.

This setting is about hiding the scrollbar, while you're not scrolling. When set to "Always", the scrollbar will not disappear, when you're not scrolling.

<img width="1178" alt="always" src="https://user-images.githubusercontent.com/37622889/158026323-815c4746-82d4-484f-82cb-928f25c574d9.png">

When set to "When scrolling", the scrollbar won't be visible as long as you're not scrolling. Additionally, the scrollbar is just layered on top of the content instead of having its own dedicated area on the side. (See the following video)

https://user-images.githubusercontent.com/37622889/158026349-cddc1d07-23f3-4a5d-992d-98d61f6e225b.mp4
